api/source/specification/stig-manager.yaml

```diff
@@ -1305,6 +1305,350 @@ paths:
+            - 'stig-manager:collection'
+    delete:
+      tags:
+        - Collection
+      summary: Remove the provided Collection metadata key
+      operationId: deleteCollectionMetadataKey
+      responses:
+        '204':
+          description: Empty to avoid large response after deleteing key with large value
+        default:
+          description: unexpected error
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Error'
+      security:
+        - oauth:
+            - 'stig-manager:collection'
+  '/assets/{assetId}/metadata':
+    parameters:
+      - $ref: '#/components/parameters/ElevateQuery'
+      - $ref: '#/components/parameters/AssetIdPath'
+    get:
+      tags:
+        - Asset
+      summary: Return the metadata for Asset
+      operationId: getAssetMetadata
+      responses:
+        '200':
+          description: Metadata response
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Metadata'
+        default:
+          description: unexpected error
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Error'
+      security:
+        - oauth:
+            - 'stig-manager:collection:read'
+    patch:
+      tags:
+        - Asset
+      summary: Merge the provided object to a Assetmetadata
+      operationId: patchAssetMetadata
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/Metadata'
+      responses:
+        '200':
+          description: Metadata response
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Metadata'
+        default:
+          description: unexpected error
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Error'
+      security:
+        - oauth:
+            - 'stig-manager:collection'
+    put:
+      tags:
+        - Asset
+      summary: Set a Asset metadata to the provided object
+      operationId: putAssetMetadata
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/Metadata'
+      responses:
+        '200':
+          description: Metadata response
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Metadata'
+        default:
+          description: unexpected error
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Error'
+      security:
+        - oauth:
+            - 'stig-manager:collection'
+  '/assets/{assetId}/metadata/keys':
+    parameters:
+      - $ref: '#/components/parameters/ElevateQuery'
```

Comment:
Please remove the `elevate` param from this endpoint

Reply:
removed